packages/extension-polkagate/src/popup/staking/solo/settings/Review.tsx

```
@@ -128,7 +128,7 @@ export default function Review({ address, api, newSettings, setRefresh, setShow,
     const tx = txs.length === 2 ? batchAll(txs) : txs[0];
 
     setTx(tx);
-    tx && tx.paymentInfo(formatted).then((i) => setEstimatedFee(api.createType('Balance', i?.partialFee ?? BN_ZERO))).catch(console.error);
+    tx && tx.paymentInfo(formatted).then((i) => setEstimatedFee(api.createType('Balance', i?.partialFee ?? BN_ZERO)) as Balance).catch(console.error);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)